.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
I'm really for not using depandabot which got proven to create more noise than solving any actual issue in practise on most asf project, is it ok to drop it?